CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Window state is saved on close and restored on start.
 - Files without a defined external file type are now directly opened with the default aplication of the operating system
 - We streamlined the process to rename and move files by removing the confirmation dialogs.
+- We removed the redundant new lines of markings and wrapped the summary in the File annotation tab. [#3823](https://github.com/JabRef/jabref/issues/3823)
```

Comment:
This comment is probably a leftover from the other PR.

Reply:
So what should I do after remove the additional changelog entry? Commit and push?

Reply:
This is my first time pull request.

Reply:
Yes, everything is fine now. Thanks! 
We will wait until a second developer gives a positive review before merge.